rest/client/methods.go

```diff
@@ -1571,3 +1571,13 @@ func (c *communicatorImpl) CompareTasks(ctx context.Context, tasks []string, use
 
 	return results.Order, results.Logic, nil
 }
+
+// FindHostByIP queries the database for the host with ip matching the ip address
+func (c *communicatorImpl) FindHostByIpAddress(ip string) (*host.Host, error) {
```

Comment:
What is the benefit of having a second copy of this function in the `client` package, instead of reusing the function in the `data` package?

Reply:
this function is currently wrong but the new version of the function uses the endpoint route with a context to return a APIHost model instead of it being identical to the function i the data package